Library/Formula/kobalt.rb

```diff
@@ -0,0 +1,38 @@
+class Kobalt < Formula
+  desc "Build system"
+  homepage "http://beust.com/kobalt"
+  url "https://github.com/cbeust/kobalt/releases/download/0.633/kobalt-0.633.zip"
+  sha256 "bb468a7b8761de20c4700e18a6de55ee0712edd0e9d04748e53592c91389c94e"
+
+  def install
+    libexec.install %w[kobaltw kobalt]
+    kobaltw = libexec/"kobaltw"
+    kobaltw.chmod 0755
+    bin.write_exec_script kobaltw
+  end
+
+  test do
+    (testpath/"src/main/kotlin/com/A.kt").write <<-EOS.undent
+      package com
+      class A
+      EOS
+
+    (testpath/"kobalt/src/Build.kt").write <<-EOS.undent
+      import com.beust.kobalt.*
+      import com.beust.kobalt.api.*
+      import com.beust.kobalt.plugin.packaging.*
+
+      val p = project {
+        name = "test"
+        version = "1.0"
+        assemble {
+        jar {}
+      }
+    }
+    EOS
+
+    system "#{bin}/kobaltw", "assemble"
```

Comment:
This is trying to write to the wrong $HOME: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/42192/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_kobalt/

Add a `ENV.java_cache` line to `test do` which should fix it.